html/cross-origin-embedder-policy/none.https.html

```diff
@@ -4,8 +4,12 @@
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script src=/common/utils.js></script> <!-- Use token() to allow running tests in parallel -->
+<script src="/common/get-host-info.sub.js"></script>
 <div id=log></div>
 <script>
+const HOST = get_host_info();
+const BASE = "/html/cross-origin-embedder-policy/resources";
```

Comment:
Let's calculate this using `new URL("resources", location).pathname` to make it easier to move things around. (Same in the other file.)

Reply:
Fixed.